node/src/BaseClient.ts

```diff
@@ -1250,6 +1251,34 @@ export class BaseClient {
         );
     }
 
+    /** Returns the rank of `member` in the sorted set stored at `key`, with scores ordered from low to high.
+     * See https://redis.io/commands/zrank for more details.
+     * To get the rank of `member` with it's score, see `zrankWithScore`.
+     *
+     * @param key - The key of the sorted set.
+     * @param member - The member whose rank is to be retrieved.
+     * @returns The rank of `member` in the sorted set.
+     * If `key` doesn't exist, or if `member` is not present in the set, null will be returned.
+     */
+    public zrank(key: string, member: string): Promise<number | null> {
+        return this.createWritePromise(createZrank(key, member));
+    }
+
+    /** Returns the rank of `member` in the sorted set stored at `key` with it's score, where scores are ordered from the lowest to highest.
+     * See https://redis.io/commands/zrank for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param member - The member whose rank is to be retrieved.
+     * @returns A list containing the rank and score of `member` in the sorted set.
+     * If `key` doesn't exist, or if `member` is not present in the set, null will be returned.
+     */
+    public zrankWithScore(
```

Comment:
This only works with Redis 7.2 and above. 
```
@since Redis version 7.2.0. 
```

Reply:
@adanWattad  please all fix Andrew's comments in python